src/coreclr/src/jit/block.h

```diff
@@ -514,16 +514,14 @@ struct BasicBlock : private LIR::Range
     const char* dspToString(int blockNumPadding = 0);
 #endif // DEBUG
 
-    typedef unsigned weight_t; // Type used to hold block and edge weights
-                               // Note that for CLR v2.0 and earlier our
-                               // block weights were stored using unsigned shorts
+    // Type used to hold block and edge weights
+    typedef float weight_t;
 
-#define BB_UNITY_WEIGHT 100 // how much a normal execute once block weights
-#define BB_LOOP_WEIGHT 8    // how much more loops are weighted
-#define BB_ZERO_WEIGHT 0
-#define BB_MAX_WEIGHT UINT32_MAX // we're using an 'unsigned' for the weight
-#define BB_VERY_HOT_WEIGHT 256   // how many average hits a BB has (per BBT scenario run) for this block
-                                 // to be considered as very hot
+#define BB_UNITY_WEIGHT 100.0f       // how much a normal execute once block weights
```

Comment:
here and below: weights => weighs (sure, pre-existing)

Reply:
Fixed.